cap/modules/workflows/views.py

```diff
@@ -109,95 +109,75 @@ def ping_reana():
 
 @workflows_bp.route('/', methods=['GET'])
 @login_required
-def get_workflows_view():
+def get_workflows_by_user():
     """Create a reana workflow by json."""
     workflows = ReanaWorkflow.get_user_workflows(current_user.id)
 
-    _workflows = [workflow.serialize() for workflow in workflows]
-    return jsonify(_workflows)
+    workflows_serialized = [workflow.serialize() for workflow in workflows]
```

Comment:
nit: you can also use ```many=True``` on your schema, then just 
```
schema = YourSchema(many=True)
result = schema.dump(workflows)
```

Reply:
that's pamfilos, so i am not touching it